meta-mender-qemu/recipes-support/libp11/libp11_0.4.%.bbappend

```diff
@@ -0,0 +1,4 @@
+DEPENDS = "libtool openssl p11-kit"
```

Comment:
This we can change with:
```suggestion
DEPENDS_append = " p11-kit"
```

But actually, I am not sure it is correct for the library to depend on the `p11-kit`. Doesn't it compile without it? (This is a build time dependency, not a runtime one).

Reply:
this is the most important part! this is what makes the library to be loaded.

Reply:
changed to _append.

Reply:
works! thanks!

Reply:
I still don't think we need it at all. I thought `DEPENDS` would only pull build time dependencies, not make the library to be added the the image (for that we have `RDEPENDS`). @kacf can you correct me?

Reply:
`DEPENDS_append = " p11-kit"` was the main mistery here! unless it is in we get a `/usr/lib/engines-1.1/pkcs11.so` that was not working (see long thread on #client-team) without this in the opnessl did not load the libs it needed and the error showed up.